src/main/java/gregtech/common/blocks/BlockNuclearCasing.java

```diff
@@ -0,0 +1,48 @@
+package gregtech.common.blocks;
+
+import gregtech.api.block.IStateHarvestLevel;
+import gregtech.api.block.VariantActiveBlock;
+
+import net.minecraft.block.SoundType;
+import net.minecraft.block.material.Material;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.util.IStringSerializable;
+
+import org.jetbrains.annotations.NotNull;
+
+public class BlockNuclearCasing extends VariantActiveBlock<BlockNuclearCasing.NuclearCasingType> {
+
+    public BlockNuclearCasing() {
+        super(Material.IRON);
+        setTranslationKey("nuclear_casing");
+        setHardness(5.0f);
+        setResistance(10.0f);
+        setSoundType(SoundType.METAL);
+        setDefaultState(getState(NuclearCasingType.SPENT_FUEL_CASING));
+    }
+
+    public enum NuclearCasingType implements IStringSerializable, IStateHarvestLevel {
+
+        SPENT_FUEL_CASING("spent_fuel_casing", 2),
+        GAS_CENTRIFUGE_HEATER("gas_centrifuge_heater", 1);
```

Comment:
why is GasCentrifugeHeater part of NuclearCasing and not GasCentrifugeCasing?

Reply:
Fixed. I think someone might have worried the not-full-blockness of the latter would mess something up?